### PR TITLE
Adjustment of format used for times in TimeSeries and a test that ver…

### PR DESCRIPTION
**Pull request by rripken**
…ifies json roundtrip serialization.

### CODE REVIEW

File: cwms_radar_api/src/test/java/cwms/radar/data/dto/TimeSeriesTest.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		ZonedDateTime start = ZonedDateTime.parse("2021-06-21T14:00:00-07:00[PST8PDT]");
+		ZonedDateTime end = ZonedDateTime.parse("2021-06-22T14:00:00-07:00[PST8PDT]");
+		TimeSeries ts = new TimeSeries(null, -1, 0, tsId, "LRL", start, end, null, Duration.ZERO);
+
+		ObjectMapper om = buildObjectMapper();
+
+		String tsBody = om.writeValueAsString(ts);
+		assertNotNull(tsBody);
+
+		TimeSeries ts2 = om.readValue(tsBody, TimeSeries.class);
```
Comment (DanielTOsborne) [inline cwms_radar_api/src/test/java/cwms/radar/data/dto/TimeSeriesTest.java:37]: The test fails on this line, with the following message:
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException at TimeSeriesTest.java:37

**Review by DanielTOsborne — COMMENTED** [review]
I do have a question about how imports should be listed, but aside from that, they appear to not be directly related to the fix (aside from the first hunk), but are the majority of the change content in the TimeSeries.java file.

It's just a nitpick though. The fix is good. 👍

File: cwms_radar_api/src/test/java/cwms/radar/data/dto/TimeSeriesTest.java (hunk 1 above)
Comment (rripken) [reply cwms_radar_api/src/test/java/cwms/radar/data/dto/TimeSeriesTest.java:37]: Oh, I fixed that too. I was trying to figure out why my IDE was listing TimeSeries as changed in two changelists.  Weird.  I'll push that too.

Comment (rripken) [reply cwms_radar_api/src/test/java/cwms/radar/data/dto/TimeSeriesTest.java:37]: If my IDE complains about unused imports I usually just let it "optimize imports" or whatever it calls it.  I don't pay too much attention to how imports are ordered.

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**